core/bootstrap.go

```diff
@@ -86,7 +86,7 @@ func bootstrap(ctx context.Context,
 
 	// we can try running dht bootstrap even if we're connected to all bootstrap peers.
 	if len(h.Network().Conns()) > 0 {
-		if err := r.Bootstrap(ctx, numDHTBootstrapQueries); err != nil {
+		if _, err := r.Bootstrap(); err != nil {
```

Comment:
yes, but if we do that, we should do that with regular supervise connections process, _thats_ what the core should have a handle to.